tools/observability/README.md

```diff
@@ -1 +1,35 @@
-[See docs here](https://docs.determined.ai)
\ No newline at end of file
+[See docs here](https://docs.determined.ai)
+
+# Metrics
+Determined does not produce metrics, rather it uses existing tools to report information. Please see source documentation for details on specific metrics.
```

Comment:
we should probably document the health metric https://github.com/determined-ai/determined/blob/43289e9afcf780ec8b0bc7695b450408d391d2bf/docs/integrations/prometheus/_index.rst?plain=1#L182